tuxemon/states/combat/combat_animations.py

```diff
@@ -152,6 +152,7 @@ def animate_monster_release(
         feet = (feet_list[0], feet_list[1] + tools.scale(11))
 
         capdev = self.load_sprite(f"gfx/items/{monster.capture_device}.png")
+        assert capdev.image
```

Comment:
Again, if `capdev.image` is undefined it should already fail earlier (`graphics.load_sprite` would fail)